taxonomy/utils.py

```diff
@@ -145,128 +95,81 @@ def process_program_skills_data(program, program_skills, should_commit_to_db):
                 'description': skill['description']
             }
             if should_commit_to_db:
-                update_program_skills_data(program['uuid'], skill_external_id, confidence, skill_data)
+                update_skills_data(
+                    course_or_program[key_or_uuid], skill_external_id, confidence, skill_data, is_programs
+                )
         except KeyError:
-            message = f'[TAXONOMY] Missing keys in skills data for program_uuid: {program["uuid"]}'
+            message = f'[TAXONOMY] Missing keys in skills data for key: {course_or_program[key_or_uuid]}'
             LOGGER.error(message)
-            failures.append((program['uuid'], message))
+            failures.append((course_or_program['uuid'], message))
         except (ValueError, TypeError):
-            message = f'[TAXONOMY] Invalid type for `confidence` in program skills for program_uuid: {program["uuid"]}'
+            message = f'[TAXONOMY] Invalid type for `confidence` in skills for key: {course_or_program[key_or_uuid]}'
             LOGGER.error(message)
-            failures.append((program['uuid'], message))
+            failures.append((course_or_program['uuid'], message))
     return failures
 
 
-def refresh_course_skills(courses, should_commit_to_db):
+def refresh_skills(courses_or_programs, should_commit_to_db, is_programs=False):
```

Comment:
Should we name this method as `refresh_product_skills` and then rather `course_or_programs` we can simple use `products` to keep it consistent with https://github.com/openedx/taxonomy-connector/pull/99/files#diff-c16bb57d10bc9185110cec11767320e1aa531630fae2c130e353f7379e032885R158